packages/@lwc/engine/src/framework/main.ts

```diff
@@ -8,6 +8,8 @@
 // Polyfills
 import '../polyfills/proxy-concat/main';
 import '../polyfills/aria-properties/main';
+
+// External libraries which require initalization
```

Comment:
I still don't fully understand why is this important since we are not caching any insertion and removal methods anymore.